cpu/stm32f1/include/cpu_conf.h

```diff
@@ -36,12 +37,122 @@ extern "C" {
 #endif
 
 /**
+ * @brief   Flash page configuration
```

Comment:
What defines do you mean specifically? I agree we could move `FW_SLOT*` defines to maybe cortex_common. Otherwise you prefer to create another .h file containing these on something like cpu/include?